Packages/com.chisel.editor/Chisel/Editor/Components/Generators/CSGLinearStairs/Editor/CSGLinearStairsGeneratorMode.cs

```diff
@@ -41,59 +41,54 @@ public void OnSceneGUI(SceneView sceneView, Rect dragArea)
 			CSGModel modelBeneathCursor;
 			Matrix4x4 transformation;
 			float height;
-			switch (BoxExtrusionHandle.Do(dragArea, out bounds, out height, out modelBeneathCursor, out transformation, false, false, Axis.Y))
+            
+            var flags = BoxExtrusionFlags.AlwaysFaceUp;
```

Comment:
BoxExtrusionHandle (need to find a better name) doesn't handle this flag yet, but this would basically allow us to build a stairs by starting the box shape on the wall, instead of the floor.
Now technically this already works fine for linear stairs, but it doesn't for spiral stairs that uses the same flag. Currently the spiral stairs would use the wall as the floor and be created sideways, when we want the spiral stairs to be pointed upwards (the most common situation after all)
This will be adressed in a future PR